Mathlib/Data/Set/Card.lean

```diff
@@ -481,6 +492,17 @@ theorem Finite.exists_bijOn_of_encard_eq [Nonempty β] (hs : s.Finite) (h : s.en
   rw [(hs.image f).eq_of_subset_of_encard_le (image_subset_iff.mpr hf)
     (h.symm.trans hinj.encard_image.symm).le]
 
+/-- A version of the pigeonhole principle for `Set`s rather than `Finset`s.
```

Comment:
Can we also have the version of this for `Set.ncard` (perhaps assuming finiteness of s or t if needed)?

Reply:
It already exists. I have updated doc strings to make each of these discoverable from the other.